app/error/RendersErrors.scala

```diff
@@ -48,13 +48,15 @@ trait RendersErrors extends Results {
       case NOT_FOUND   => "pageNotFound404"
       case _           => "InternalServerError500"
     }
-
-    Status(statusCode)(
-      views.html.error(
-        s"global.error.$errorKey.title",
-        Some(s"global.error.$errorKey.heading"),
-        List(s"global.error.$errorKey.message")))
-
+    if (statusCode == NOT_FOUND) {
```

Comment:
I think the only time this is possible is in LocalErrorHandler, where we call futureError(statusCode) on line 47. 

You just need to check if the status code is NOT_FOUND in there and render the correct view.